include/libswiftnav/signal.h

```diff
@@ -15,68 +16,85 @@
 
 #include <libswiftnav/common.h>
 
+/** \addtogroup signal
+ * \{ */
+
+/* Number of satellites in each constellation. */
 #define NUM_SATS_GPS 32
 #define NUM_SATS_SBAS 19
 #define NUM_SATS (NUM_SATS_GPS + NUM_SATS_SBAS)
 
+/* Number of codes in each constellation. */
+#define NUM_CODES_GPS 2
```

Comment:
These are kind of redundant and never used if NUM_SIGNALS_<code> defines are used. Although I am a little bit iffy on those defines since is there ever a situation where they don't equal NUMS_SATS_<constellation> values?